### PR TITLE
content/recruit-cn: update positions

### PR DESCRIPTION
**Pull request by Soline324**
1. add a new position: administrative receptionist;
2. modify other positions
@xuechunL PTAL

### CODE REVIEW

File: content/recruit-cn/engineering/bizdev-cloud-engineer.md [hunk 1]
```diff
@@ -1,49 +1,49 @@
 ---
 title: TiDB 商业产品开发 - Cloud 方向
 tags:
-  - Engineering
-location: 北京，上海，广州，杭州
+  - Tech
```
Comment (xuechunL) [inline content/recruit-cn/engineering/bizdev-cloud-engineer.md]: Maybe use `engineer` is better,

Comment (Soline324) [reply content/recruit-cn/engineering/bizdev-cloud-engineer.md]: Ok

File: content/recruit-cn/general-administrative/administrative-receptionist.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: 行政前台
+tags:
+  -  'General & Administrative'
+location: 北京
+avatar: receptionist
```
Comment (xuechunL) [inline content/recruit-cn/general-administrative/administrative-receptionist.md]: Before this PR is merged I need to add an icon for the type of receptionist in the code first.

**Comment by Soline324** [comment]
@xuechunL 
The tags of engineering positions has been changed into "Engineering" from "Tech".
PTAL

**Pull request merged**